modules/repo_sync/devportal_repo_sync.libraries.yml

```diff
@@ -0,0 +1,5 @@
+devportal-repo-sync-import-page:
```

Comment:
I think this should be simply "import-page" or "import_page". When you attach a library to something you have to specify the module name anyway (for example: devportal_repo_sync/import_page) so prefixing a library name with the module name is unnecessary.